src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py

```diff
@@ -1972,7 +1972,7 @@ def test_keyvault_certificate_crud(self, resource_group, key_vault):
         self.cmd('keyvault certificate delete --vault-name {kv} -n cert1')
         time.sleep(60)
         self.cmd('keyvault certificate purge --vault-name {kv} -n cert1')
-        time.sleep(10)
+        time.sleep(120)
```

Comment:
Is this change related to this PR?

Reply:
No it's for debug, I have reverted this change.